client/src/components/App.jsx

```diff
@@ -51,6 +53,10 @@ class _App extends React.Component {
     let homeRedirect
     let navbar
 
+    if (user.error != null) {
+      this._userLogout()
+    }
```

Comment:
Does this mean that the user will always be logged out unless we explicitly set an error on the user state?

Reply:
There's only 2 cases atm where a dispatched action would result in an error in the user state:
1.) if a login attempt fails
2.) if the check token + id fails on a previously logged in user state

If one of those events occur, user.error will be set and the app will reset the user state and redirect to login.